projects/plugins/crm/.gitignore

```diff
@@ -1,37 +1,20 @@
-.DS_STORE
-.idea/
-.phpcs.xml
-*.DS_Store
-**/.DS_Store
-/.DS_STORE
-/build/
-/node_modules/
-/vendor/
-codeception.yml
-**/css/*.min.rtl.*
-**/css/**/*.min.rtl.*
-**/css/*.rtl.css
-**/css/**/*.rtl.css
-**/css/*.min.css
-**/css/welcome-to-zbs/*.min.css
+vendor/
+node_modules/
+build/
+.cache/
+**/css/*.css 
+**/css/**/*.css
 **/css/integrations/*
-**/css/font-awesome.css
+**/js/*.min.js.map
+**/js/**/*.min.js.map
 **/js/*.min.js
 **/js/welcome-to-zbs/*.min.js
 **/js/*.min.js.LICENSE.txt
-.cache/
-phpcs.xml
-phpcs.security.xml
```

Comment:
These are overrides for the distribution `.xml`, so we should probably leave them (or remove PHPCS config from the repo altogether).

Reply:
Not sure I follow. i couldn't find the files locally - when you say these are overrides does that mean these might be generated in some situations?

Reply:
Ah just saw your follow-on comment below, will follow up in another comment.